packages/launch/src/hooks/use-domain-search.ts

```diff
@@ -23,7 +23,7 @@
```

Comment:
TS issue here because `title` can also be `undefined`. I guess one way to fix it is just go `title || ''` or to allow `filterUnsuitableTitles` to accept `undefined`. However it's done I think we just need to keep the existing behaviour of going into the `if ( ! search || ... )` branch if title is `undefined`.